### PR TITLE
Update to 1.2 release of Wicked Good XPath.

### PR DESCRIPTION
**Pull request by Dominator008**

### CODE REVIEW

File: wgxpath.install.js [hunk 1]
```diff
@@ -27,65 +27,50 @@ function q(a){var b=r;function c(){}c.prototype=b.prototype;a.G=b.prototype;a.pr
...
-0>a?null:c[a]}}Y.ANY_TYPE=0;Y.NUMBER_TYPE=1;Y.STRING_TYPE=2;Y.BOOLEAN_TYPE=3;Y.UNORDERED_NODE_ITERATOR_TYPE=4;Y.ORDERED_NODE_ITERATOR_TYPE=5;Y.UNORDERED_NODE_SNAPSHOT_TYPE=6;Y.ORDERED_NODE_SNAPSHOT_TYPE=7;Y.ANY_UNORDERED_NODE_TYPE=8;Y.FIRST_ORDERED_NODE_TYPE=9;function Qb(a){this.lookupNamespaceURI=Za(a)}
-function Rb(a){a=a||m;var b=a.document;b.evaluate||(a.XPathResult=Y,b.evaluate=function(a,b,e,f){return(new Pb(a,e)).evaluate(b,f)},b.createExpression=function(a,b){return new Pb(a,b)},b.createNSResolver=function(a){return new Qb(a)})}var Sb=["wgxpath","install"],Z=m;Sb[0]in Z||!Z.execScript||Z.execScript("var "+Sb[0]);for(var Tb;Sb.length&&(Tb=Sb.shift());)Sb.length||void 0===Rb?Z[Tb]?Z=Z[Tb]:Z=Z[Tb]={}:Z[Tb]=Rb;})()
-
-// Export for Node.js.
```
Comment (jacobmarble) [inline wgxpath.install.js]: This section cannot be omitted.

Comment (jacobmarble) [inline wgxpath.install.js:77]: This section cannot be omitted.

**Comment by jacobmarble** [comment]
Thanks for your hard work. I've noticed the upstream patches you have submitted in the last few days.

Please test this PR using the example mentioned in the README.

File: wgxpath.install.js (hunk 1 above)
Comment (Dominator008) [inline wgxpath.install.js:77]: Done.

**Comment by Dominator008** [comment]
Tested against `word_of_the_day.js` with an updated XPath reflecting the change of http://www.merriam-webster.com/word-of-the-day.

**Pull request merged**